.github/workflows/ci.yml

```diff
@@ -4,13 +4,13 @@ on: [push]
 
 jobs:
   build: 
-    name: build-wasm
+    name: build-linux
     runs-on: ubuntu-latest
-    container: nxxm/nxxm-ubuntu
+    container: tipibuild/tipi-ubuntu
     steps:
       - name: checkout
         uses: actions/checkout@v2
-      - name: nxxm builds project 
+      - name: tipi builds project 
         run: |
           export HOME=/root
-          nxxm . --verbose --test all 
+          tipi . -t linux-cxx17 --verbose
```

Comment:
No tests are run anymore here

Reply:
I mean there is an example/ ok, I thought there were tests, so it seems we don't have any tests for the feature. We should add some to our e2e test suite.  Possibly we can turn the example in an unit test for this.